package.json

```diff
@@ -47,6 +47,7 @@
     "bcryptjs": "^2.4.3",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.1",
+    "dotenv": "^16.4.5",
```

Comment:
Can this be taken out? We have `@nestjs/config` to handle environment variables.

Reply:
done @CFCIfe